Translations update from Kodi Weblate

weblate opened the pull request:
Translations update from [Kodi Weblate](https://kodi.weblate.cloud) for [Kodi remotes/Kore](https://kodi.weblate.cloud/projects/kodi-remotes/kore/).



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widget/kodi-remotes/kore/horizontal-auto.svg)

Danny3 commented:
Can you please do it again?
And then merge + release a new version of the app so we can test the translations?
Thanks!

Danny3 commented:
Too bad Kore is never getting any new release...
So testing the translations is impossible!

weblate committed:
Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Galician (Spain) (gl_es))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (346 of 346 strings)

Deleted translation using Weblate (Ossetian (os_os))

Deleted translation using Weblate (Occitan (France) (oc_fr))

Deleted translation using Weblate (Portuguese (Brazil))

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 98.8% (342 of 346 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 98.8% (342 of 346 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (346 of 346 strings)

Co-authored-by: Alexey <signfinder@gmail.com>
Co-authored-by: Alfonso Cachero <alfonso.cachero@gmail.com>
Co-authored-by: Daniel <danny3@tutanota.com>
Co-authored-by: Elvis Gallegos <elvis.gallegos@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Hugoren Martinako <aumpfbahn@gmail.com>
Co-authored-by: Kai Sommerfeld <ksooo@users.noreply.kodi.weblate.cloud>
Co-authored-by: Linerly <linerly@proton.me>
Co-authored-by: Oskari Lavinto <olavinto@protonmail.com>
Co-authored-by: Pavlo Marianov <acid@jack.kyiv.ua>
Co-authored-by: rimasx <riks_12@hot.ee>
Co-authored-by: skypichat <skypichat@hotmail.fr>
Co-authored-by: wabisabi926 <liwenliang926@163.com>
Co-authored-by: wabisabi926 <wabisabi926@foxmail.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/ca_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/es_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/fr_fr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/gl_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/id_id/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/ro_ro/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/uk_ua/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/zh_Hans/
Translation: Kodi remotes/Kore

weblate committed:
Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 99.7% (345 of 346 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Galician (Spain) (gl_es))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (346 of 346 strings)

Deleted translation using Weblate (Ossetian (os_os))

Deleted translation using Weblate (Occitan (France) (oc_fr))

Deleted translation using Weblate (Portuguese (Brazil))

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 98.8% (342 of 346 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 98.8% (342 of 346 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (346 of 346 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (346 of 346 strings)

Co-authored-by: Alexey <signfinder@gmail.com>
Co-authored-by: Alfonso Cachero <alfonso.cachero@gmail.com>
Co-authored-by: Daniel <danny3@tutanota.com>
Co-authored-by: Elvis Gallegos <elvis.gallegos@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Hugoren Martinako <aumpfbahn@gmail.com>
Co-authored-by: Kai Sommerfeld <ksooo@users.noreply.kodi.weblate.cloud>
Co-authored-by: Linerly <linerly@proton.me>
Co-authored-by: Michal Várady <miko.vaji@gmail.com>
Co-authored-by: Oskari Lavinto <olavinto@protonmail.com>
Co-authored-by: Pavlo Marianov <acid@jack.kyiv.ua>
Co-authored-by: rimasx <riks_12@hot.ee>
Co-authored-by: skypichat <skypichat@hotmail.fr>
Co-authored-by: wabisabi926 <liwenliang926@163.com>
Co-authored-by: wabisabi926 <wabisabi926@foxmail.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/ca_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/cs_cz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/es_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/fr_fr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/gl_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/id_id/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/ro_ro/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/uk_ua/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-remotes/kore/zh_Hans/
Translation: Kodi remotes/Kore
